projectDependencyGraph.gradle

```diff
@@ -40,11 +42,25 @@ task projectDependencyGraph {
         javaProjects.add(project)
       }
 
+      def folders = [new File(project.projectDir, "src")]
+      while (!folders.isEmpty()) {
+        def folder = folders.remove(0)
+        for (file in folder.listFiles()) {
+          if (file.isDirectory()) {
+            folders.add(file)
+          } else if (file.name.endsWith(".kt")) {
+            kotlinUsage.put(project, kotlinUsage.getOrDefault(project, 0) + 1)
+          } else if (file.name.endsWith(".java")) {
```

Comment:
Can this just be an `else`? I'm thinking about the case of mixed javascript and kotlin.